debian/clearwater-cluster-manager.init.d

```diff
@@ -104,7 +104,8 @@ do_start()
   etcd_cluster_key=""
   log_level=3
   log_directory=/var/log/clearwater-cluster-manager
-  
```

Comment:
I almost want this to become a breaking change.
That blank line was special, ya know. You can't just delete it willy-nilly